### ISSUE TITLE
Feature request : Brainpool ECC support

### ISSUE THREAD

**Issue by Sami32** [open-keychain/open-keychain#2898]
It would be great to be able to use our Nitrokey Brainpool generated keys:
https://docs.nitrokey.com/start/windows/openpgp-keygen-on-device
https://docs.nitrokey.com/pro/faq

https://www.rfc-editor.org/rfc/rfc5639
https://datatracker.ietf.org/doc/html/rfc6932

**Comment by srleonbr** [comment]
Enjoy and also add support for ed448, please.

**Comment by akac97** [comment]
yes please try add both brainpool and ed488 becose nist curves are backdoors and ed25519 are less safe than ed448 or brainpool
